starport/pkg/cosmosgen/templates/vuex/loader.ts.tpl

```diff
@@ -0,0 +1,27 @@
+{{ range . }}import {{ .Name }} from './{{ .Path }}'
```

Comment:
@clockworkgr I removed the [loader](https://github.com/clockworkgr/codegenui/blob/main/vue/src/store/chain/clockworkgr/codegenui/clockworkgr.codegenui.blog/index.js )inside every mode and moved loading logic into `chain/index` to simplify.

Reply:
I'll check the generated stuff to make sure but LGTM